sunpy/util/logger.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from astropy.logger import AstropyLogger
+
+
+def _init_log(config=None):
+    """Initializes the SunPy log--in most circumstances this is called
+    automatically when importing sunpy.
+    """
+
+    global log
```

Comment:
I assume this function is modified from astropy? Do we need this global?

Reply:
It is copied directly from them. I'm not exactly sure how this all works so don't want to change it.

Reply:
Well we are not referencing `log` outside of the scope of this function, so it's safe to remove.

Also if we have copied this from astropy we have to credit them and mention the licence in `licences/ASTROPY.rst`.